account/src/main/java/com/caldevsupplychain/account/service/AccountService.java

```diff
@@ -18,4 +19,6 @@
 	Optional<UserBean> findByEmailAddress(String emailAddress);
 
 	Optional<UserBean> findByToken(String token);
+
+	Optional<List<UserBean>> getAllUsers();
```

Comment:
No need to wrap `Optional` around a list. `Opitonal` is mostly use to avoid NPE and make the code more readable. NPE should not happen to a list if a list is initiated properly(`list.isEmpty()` is the equivalent of `null` for a list)

Reply:
Generated UserMapperImpl.java file in account -> builder folder is possible to return a null. 

 @Override
    public List<UserBean> usersToBeans(List<User> user) {
        if ( user == null ) {
            return null;
        }

        List<UserBean> list = new ArrayList<UserBean>( user.size() );
        for ( User user1 : user ) {
            list.add( toBean( user1 ) );
        }

        return list;
    }

Therefore I need to wrap it with an Optional.of, when I did not wrap this with Optional, it will raise an arrow.

Reply:
looks like it returns null only if the input is null. Which shouldn't happen. If users is null, we should do `users = new ArrayList<User>()` to avoid passing null to this method.